pkg/cloudevents/datacodec/xml/observability.go

```diff
@@ -0,0 +1,54 @@
+package xml
+
+import (
```

Comment:
These files seem to be mostly copies with one word sed'ed. Should it be just one, with template constructor/

Reply:
I am using the strong type in golang... I will look at consolidating